react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -1,20 +1,89 @@
-import React from 'react';
+import React, { Component } from 'react';
 
+import calculateWinner from './utils';
 import styles from './styles.module.scss';
 import Board from './components/Board';
+import MoveButton from './components/MoveButton';
 
-function Game () {
-  return (
-    <div className={styles.game}>
-      <div className={styles.gameBoard}>
-        <Board />
-      </div>
-      <div className={styles.gameInfo}>
-        <div>{/* status */}</div>
-        <ol>{/* TODO */}</ol>
+class Game extends Component {
+  state = {
+    history: [{
+      squares: Array(9).fill(null)
+    }],
+    xIsNext: true,
+    stepNumber: 0
+  };
+
+  nextTurn = () => this.state.xIsNext ? 'X' : 'O'
+
+  handleClick = (i) => {
+    const history = this.state.history.slice(0, this.state.stepNumber + 1);
+    const current = history[history.length - 1];
+    const squares = current.squares.slice();
+
+    if (!calculateWinner(squares) && !squares[i]) {
+      squares[i] = this.nextTurn();
+      this.setState(prevState => ({
+        history: history.concat([{
+          squares
+        }]),
+        xIsNext: !prevState.xIsNext,
+        stepNumber: history.length
+      }));
+    }
+  }
+
+  handleJumpTo = (step) => {
+    this.setState({
+      stepNumber: step,
+      xIsNext: step % 2 === 0
+    });
+  }
+
+  renderMoveButton = (_, move) => {
```

Comment:
Why are you using `_` on the first param?

Reply:
Because he doesn't need the first param, he needs the index